### DISCUSSION TITLE
Alert list panel instance count shows all instances, not just alerting ones

### DISCUSSION THREAD

**Discussion by erhudy** [grafana/grafana#48469] [Alerting]
**What happened**: When creating an alert list panel for a multidimensional alert query, the number of instances shown is the total number of instances, not just the ones in alerting state (see attached screenshot).

**What you expected to happen**: Ideally the number would indicate something like "1 instance alerting, 10 instances normal", "1/11 instances alerting", "10/11 instances normal", or something of that nature, so that you know how many are actually alerting before expanding the alert.

**How to reproduce it (as minimally and precisely as possible)**:

1. Create multidimensional alert (in my case alerting on scheduling issues in EKS clusters).
2. Break one of them.
3. Observe instance count on alert list panel.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.4.3
- Data source type & version: Thanos, 0.25.0
- OS Grafana is installed on: Linux
- User OS & Browser: Windows, Edge 100
- Grafana plugins: Only core
- Others: N/A
- 
<img width="507" alt="Screen Shot 2022-04-28 at 16 35 36" src="https://user-images.githubusercontent.com/1065271/165841773-86e947bd-a289-4c69-becc-6d2002a04fb0.png">

**Comment by mellieA** [comment]
@erhudy thank you for submitting this issue! We've migrated feature requests to discussions, so I've moved this one over for you and applied labels.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
